portfolio/src/main/webapp/style.css

```diff
@@ -165,6 +165,29 @@ a:hover{
     color: orangered;
 }
 
+/* Comment Formm Styling */
+label::after {
+    content: "\a";
```

Comment:
Is `\a` a line break character? If so, it might be clearer to use another CSS rule (like `display: block` on the label) to accomplish this.

Reply:
I was going for a line break. I'll try out display: block or some other CSS rule instead to make things clearer.